test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1166,7 +1167,7 @@ def test_cgroup_enforce_memory(self):
                              max_attempts=20)
         self.wait_and_remove_file(filename=o.split(':')[1], mom=self.momA)
 
-    def Xtest_cgroup_enforce_memsw(self):
+    def test_cgroup_enforce_memsw(self):
```

Comment:
Please skip this test if resources_available.vmem is not present as jobs will not run in that case.

Reply:
Here is the example,

pbsroot@testdev-09-s12:~/cgroup_dev/tests/functional> qsub -l select=1:ncpus=1:mem=300mb:vmem=320mb:host=testdev-09-s12 -- /bin/sleep 10
63.testdev-09-s12

comment = Can Never Run: Insufficient amount of resource: vmem (R: 320mb A:
	 0kb T: 0kb)

pbsroot@testdev-09-s12:~/cgroup_dev/tests/functional> pbsnodes -a | grep vmem
     resources_available.vmem = 0kb
     resources_assigned.vmem = 0kb

Reply:
I updated the code with following and added a skip
if have_swap() == 0 or not (self.server.status(NODE,'resources_available.vmem',id=self.momA)):

Reply:
You must have manually added "vmem" to the sched_config "resources:" line. The test harness doesn't do that, so the scheduler doesn't consider vmem when scheduling jobs.

Reply:
OK, now I understand why. I'll update it.